.travis.yml

```diff
@@ -13,14 +13,15 @@ env:
 before_script:
 - gem install  --no-rdoc --no-ri 'brakeman' 'bundler-audit' 'pandoc-ruby'
 - nvm install node 9.10.0
-- npm install -g yarn@1.5.1
+- npm install -g yarn
```

Comment:
Latest yarn is 1.12.3 - I'm not aware of a need to to lock it to a really old version so I changed this